opm/simulators/wells/WellInterface_impl.hpp

```diff
@@ -1396,6 +1396,70 @@ namespace Opm
         }
     }
 
+    template<typename TypeTag>
+    bool
+    WellInterface<TypeTag>::
+    wellUnderZeroRateTargetVersion(const Simulator& ebos_simulator,
```

Comment:
I think you can use `wellUnderZeroRateTargetVersion` to replace the original `wellUnderZeroRateTarget`